<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_tchmanager_detail</class>
 <widget class="QMainWindow" name="admin_tchmanager_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtQlas Alpha</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="tid_displayer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_edit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="school_edit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)
</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lesson_table">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget 
{background-color: transparent;}
QScrollBar:vertical {
    background: url(images/pure_lite_gray.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_lite_gray.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: Courier;
font-size:12px;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>506</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/admin_stmanager.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="recommendation">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>300</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>120</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/gray_minus.png)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lesson_chooser_edit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray.png)</string>
     </property>
     <property name="placeholderText">
      <string>Type to filter...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_lesson_button">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/login_plus.png)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>delete_button</zorder>
    <zorder>lesson_chooser_edit</zorder>
    <zorder>add_lesson_button</zorder>
    <zorder>recommendation</zorder>
   </widget>
   <zorder>frame</zorder>
   <zorder>tid_displayer</zorder>
   <zorder>name_edit</zorder>
   <zorder>school_edit</zorder>
   <zorder>lesson_table</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
